initca/initca.go

```diff
@@ -220,3 +219,17 @@ var CAPolicy = func() *config.Signing {
 		},
 	}
 }
+
+// CopyAndResign copies the CA certificate, updates the NotBefore and
+// NotAfter fields, and then re-signs the certificate.
+func CopyAndResign(ca *x509.Certificate, priv crypto.Signer) (cert []byte, err error) {
```

Comment:
Re-issue? Resign is kinda... negative

Reply:
Or just renew?

Reply:
We're also not issuing anything, really. Renamed to update. Now we wait for the tests to re-run.